frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/viewport-feature-manager.ts

```diff
@@ -47,10 +47,16 @@ export class ViewportFeatureManager
             olMap,
             layer,
             (targetPositions, viewport) => {
+                // We need at least on targetPosition
+                if (targetPositions.length < 1) {
+                    throw new Error(
+                        'Got unexpectedly empty targetPositions array'
+                    );
+                }
```

Comment:
I'm unsure about these. On the one hand, this is defensive programming (bad practice -> Prof. Baudisch either PT1 or PT2). On the other hand, the contract defined by OpenLayers is not clearly documented anywhere.

I tend a bit more to just keep it as before.

Reply:
I remove the check.